server/routes/visitJourney/checkYourBooking.test.ts

```diff
@@ -0,0 +1,281 @@
+import type { Express } from 'express'
+import request from 'supertest'
+import { SessionData } from 'express-session'
+import * as cheerio from 'cheerio'
+import { VisitSessionData } from '../../@types/bapv'
+import VisitSessionsService from '../../services/visitSessionsService'
+import AuditService from '../../services/auditService'
+import { appWithAllRoutes, flashProvider } from '../testutils/appSetup'
+import { SupportType, Visit } from '../../data/visitSchedulerApiTypes'
+import config from '../../config'
+import NotificationsService from '../../services/notificationsService'
+
+jest.mock('../../services/visitSessionsService')
+jest.mock('../../services/auditService')
+
+let sessionApp: Express
+const systemToken = async (user: string): Promise<string> => `${user}-token-1`
+const auditService = new AuditService() as jest.Mocked<AuditService>
+
+let flashData: Record<'errors' | 'formValues', Record<string, string | string[]>[]>
+let visitSessionData: VisitSessionData
+
+const availableSupportTypes: SupportType[] = [
```

Comment:
This data is duplicated in a several places (more now!). To be extracted out in a future PR...